blog/templates/blog/admin/post_list.html

```
@@ -0,0 +1,635 @@
+        </div>
+    </div>
+
+    <h2 id="post-list-section">게시물 목록</h2>
+    <div class="post-list">
+        <form method="get" class="filter-box" id="post-search-form">
+            <input type="text" name="search" placeholder="검색" value="{{ search }}">
+
+            <select name="category">
+                <option value="">전체 카테고리</option>
+                {% for c in categories %}
+                <option value="{{ c.id }}" {% if category_id == c.id|stringformat:"s" %}selected{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)